CodeRater/CodeRater.cpp

```diff
@@ -46,17 +46,24 @@ void CodeRater::checkKraftsInequality() {
     }
 
     if (kraftsSum < 1) {
-        std::cout << "│  Code is not optimal:\t" << kraftsSum << " < 1" << std::endl;
+        //std::cout << "│  Code is not optimal:\t" << kraftsSum << " < 1" << std::endl;
```

Comment:
You should never leave comments in your code. You can always use git to obtain history.